sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/api/Azure.AI.Language.QuestionAnswering.netstandard2.0.cs

```diff
@@ -188,6 +188,57 @@ public partial class TextDocument
         public string Text { get { throw null; } }
     }
 }
+namespace Azure.AI.Language.QuestionAnswering.Projects
+{
+    public partial class QuestionAnsweringProjectsClient
+    {
+        protected QuestionAnsweringProjectsClient() { }
+        public QuestionAnsweringProjectsClient(System.Uri endpoint, Azure.AzureKeyCredential credential, Azure.AI.Language.QuestionAnswering.QuestionAnsweringClientOptions options = null) { }
+        public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
+        public virtual Azure.Response AddFeedback(string projectName, Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response> AddFeedbackAsync(string projectName, Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Response CreateProject(string projectName, Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response> CreateProjectAsync(string projectName, Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Response DeleteProject(string projectName, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response> DeleteProjectAsync(string projectName, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Operation<System.BinaryData> DeployProject(string projectName, string deploymentName, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Operation<System.BinaryData>> DeployProjectAsync(string projectName, string deploymentName, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Operation<System.BinaryData> Export(string projectName, string format = null, string assetKind = null, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Operation<System.BinaryData>> ExportAsync(string projectName, string format = null, string assetKind = null, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Response GetDeleteStatus(string jobId, Azure.RequestContext context = null) { throw null; }
```

Comment:
@annelo-msft @pakrym should raw status getters be added, or was this because the LRO method wasn't marked explicitly as an LRO? https://github.com/Azure/azure-rest-api-specs/blob/f3c2b263da9afc93edb71d55d9b08c75de7401d9/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json#L187-L194

Though, I do see methods like `GetExportStatus` below where that method is marked as an LRO. If that's expected, that's fine. I just thought maybe we'd generate an `Operation<T>` or is that considered too high-level?

/cc @rokulka @annatisch @mshaban-msft

Reply:
Still, @rokulka, any reason delete is not marked as an LRO while import, export, deploy, and update (sources, QnAs) are? Delete is returning an `Operation-Location` header. I also notice the job status endpoint is different from the others: https://github.com/Azure/azure-rest-api-specs/blob/f3c2b263da9afc93edb71d55d9b08c75de7401d9/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json#L209

Reply:
I ping @rokulka offline (with you and @annatisch) as well on this one. If it's a quick turnaround (fix or no fix), let's wait to get it in this PR; otherwise, if some time is required feel free to have this one merged, open a tracking issue on this potential issue, and we can get it in another PR.

Reply:
@heaths  @annelo-msft  @pakrym - Project Delete is also an LRO. Sorry, I missed adding the property in the swagger - this API was a last minute addition.  The Operation-Location URL returned is for the path /deletion-jobs/{jobId} .

Reply:
What timeframe do you have for pushing a change? Otherwise, @AhmedLeithy you can use a swagger transform. Talk to @tg-msft in my stead if you need help.

Reply:
Let's make sure we update the swagger document if there is missing information in there. We should avoid transforms that only exist because we didn't get around to updating the swagger itself.

Reply:
@johanste agreed, but *temporary* transforms can be useful to unblock. We still need the swagger change, yes. But it looks like Azure/azure-rest-api-specs#16832 is close to getting merged to main.

Reply:
Offline, @AhmedLeithy said the changes were merged a while back and necessary changes herein.